### PR TITLE
Releasing version v0.11.0

### PR DESCRIPTION
**Pull request by vishh**
I will make a github release once this PR is merged.

### CODE REVIEW

File: RELEASES.md [hunk 1]
```diff
@@ -1,25 +1,30 @@
 # Release Notes
 
-## 0.11.0 (Unreleased)
+## 0.11.0 (4-28-2015)
 - Export filesystem usage metrics for the node and containers.
+- Support for Kubernetes events
+- New Sink - Google Cloud Logging. Supports events only.
+- New Sink - Google Cloud Monitoring. Supports metrics only.
+- New metric labels - 'pod_name' and 'resource_id'.
+- Extensible source and sinks configuration. It is now possible to multiple sources and sinks.
```
Comment (vmarmol) [inline RELEASES.md]: possible to export to?

Comment (vishh) [inline RELEASES.md]: Good catch :) Fixed

File: RELEASES.md [hunk 2]
```diff
@@ -1,25 +1,30 @@
 # Release Notes
 
-## 0.11.0 (Unreleased)
+## 0.11.0 (4-28-2015)
 - Export filesystem usage metrics for the node and containers.
+- Support for Kubernetes events
+- New Sink - Google Cloud Logging. Supports events only.
+- New Sink - Google Cloud Monitoring. Supports metrics only.
+- New metric labels - 'pod_name' and 'resource_id'.
+- Extensible source and sinks configuration. It is now possible to export to multiple sources and sinks.
```
Comment (vmarmol) [inline RELEASES.md]: I think you mean it is now possible to have multiple sources and sinks. We don't export to sources :)

File: RELEASES.md (hunk 1 above)
Comment (vishh) [inline RELEASES.md]: I am getting ahead of myself! Multiple sources isn't supported yet. Fixed

**Comment by vmarmol** [comment]
LGTM

**Comment by vmarmol** [comment]
Will let the CI's run before merge to make sure the release will be green :)

**Comment by vishh** [comment]
I am considering switching over to some other CI service. Travis is too slow.

**Comment by vishh** [comment]
I ran the test manually and they pass. I think we can go ahead and merge this PR since the only code change is that of updating the version number.

**Comment by vmarmol** [comment]
@vishh Travis is dead today (as is Shippable :( ), but Jenkins runs everything so we're good!

**Comment by vishh** [comment]
@vmarmol: Ah! I wasn't aware of the outage. I am thinking of adding support for CircleCI just as a backup. WDYT?

**Comment by vmarmol** [comment]
Sure :D

**Pull request merged**